javaslang-test/src/main/java/javaslang/test/Arbitrary.java

```diff
@@ -197,6 +199,18 @@
         return size -> random -> Gen.choose(-size, size).apply(random);
     }
 
+    static Arbitrary<LocalDateTime> localDateTime() {
+        return localDateTime(ChronoUnit.DAYS);
+    }
+
+    static Arbitrary<LocalDateTime> localDateTime(ChronoUnit unit) {
+        return localDateTime(LocalDateTime.now(), unit);
+    }
+
+    static Arbitrary<LocalDateTime> localDateTime(LocalDateTime median, ChronoUnit unit) {
+        return size -> random -> Gen.choose(median.minus(size, unit), median.plus(size, unit)).apply(random);
+    }
+
```

Comment:
My first thought was why we are not returning the generator directly? I think the min/max bounds need to be calculated only once. I see that I did the same for Arbitrary.integer(). But I think it does not make sense...

```java
return size -> Gen.choose(median.minus(size, unit), median.plus(size, unit));
```